python/lsst/ts/electrometer/mock_server.py

```diff
@@ -221,22 +256,52 @@ def do_stop_storing_buffer(self):
 
     def do_read_buffer(self):
         """Read the values in the buffer."""
-        return "1 0 0 c\n0 0 0 c"
+        return "+0.01DC 0.33\n+0.01DC 0.33\n+0.01DC 0.33\n+0.01DC 0.33\n"
 
     def do_read_sensor(self):
+        """Read the sensor."""
         return "0"
 
     def do_get_avg_filter_status(self):
+        """Get the status of the filter in average mode."""
         return "0"
 
     def do_get_med_filter_status(self):
+        """Get the state of the filter in median mode."""
         return "1"
 
     def do_integration_time(self):
+        """Set the integration time setting."""
         return ""
 
     def do_get_integration_time(self):
+        """Get the integration time setting."""
         return "0.01"
 
     def do_get_range(self):
+        """Get the range setting."""
         return "0.1"
+
+    def do_rtc_time(self):
+        """Switch to RTClock mode."""
+        pass
+
+    def do_change_nplc(self, nplc):
+        """Change the number of programmable logic cycles
+
+        Parameters
+        ----------
+        nplc : `int`
+            The number of cycles.
+        ."""
+        pass
+
+    def do_change_sync(self, sync):
+        """Change the line synchronization setting.
+
+        Parameters
+        ----------
+        sync : `str`
```

Comment:
Is this truly a str? It sounds like a boolean from the brief description on the next line. 

If it is a str then please document its format.